HISTORY.md

```diff
@@ -52,8 +55,13 @@
 
 ### C++ API
 * Add new api,  to get a query's config. [#2167](https://github.com/TileDB-Inc/TileDB/pull/2167)
-* Removes non-default parameter in Config::unset. [#2099](https://github.com/TileDB-Inc/TileDB/pull/2099)
+* Addition of  to directly assign a config to an array. [#2142](https://github.com/TileDB-Inc/TileDB/pull/2142)
+* Addition of tiledb_array_set_config to directly assign a config to an array. [#2142](https://github.com/TileDB-Inc/TileDB/pull/2142)
```

Comment:
Here you mean `Array::set_config`. Also we added an `Array` constructor that accepts the config, as well as a config getter (if we want to be complete).